.travis.yml

```diff
@@ -40,12 +40,18 @@ jobs:
 
     - stage: release
       if: branch = release and type = push
-      script: skip
+      language: node_js
+      node_js: lts/*
+      script:
+        # Install all dependencies required for `semantic-release`
+        - npm install @semantic-release/commit-analyzer
+        - npm install @semantic-release/release-notes-generator
+        - npm install @semantic-release/changelog
+        - npm install @semantic-release/github
```

Comment:
Is this really needed when we run semantic-release with npx?

Reply:
Unfortunately yes. They are not direct dependencies of `semantic-release`.

I've tested this in a nodejs container.